docs/source/how-to-cache.mdx

```diff
@@ -0,0 +1,238 @@
+        │   └── [1.4K]  d7edf6bd2a681fb0175f7735299831ee1b22b812
+        ├── [  96]  refs
+        │   └── [  40]  main
+        └── [ 128]  snapshots
+            ├── [ 128]  2439f60ef33a0d46d85da5001d52aeda5b00ce9f
+            │   ├── [  52]  README.md -> ../../blobs/d7edf6bd2a681fb0175f7735299831ee1b22b812
+            │   └── [  76]  pytorch_model.bin -> ../../blobs/403450e234d65943a7dcf7e05a771ce3c92faa84dd07db4ac20f592037a1e4bd
+            └── [ 128]  bbc77c8132af1cc5cf678da3f1ddf2de43606d48
+                ├── [  52]  README.md -> ../../blobs/7cb18dc9bafbfcf74629a4b760af1b160957a83e
+                └── [  76]  pytorch_model.bin -> ../../blobs/403450e234d65943a7dcf7e05a771ce3c92faa84dd07db4ac20f592037a1e4bd
+```
+
+## Scan your cache
+
+At the moment, cached files are never deleted from your local directory: when you download
+a new revision of a branch, previous files are kept in case you need them again.
+Therefore it can be useful to scan your cache directory in order to know which repos
+and revisions are taking the most disk space. `huggingface_hub` provides an helper to
+do so that can be used via `huggingface-cli` or in a python script.
+
+### From the terminal
+
+The easiest way to scan your HF cache-system is to use the `scan-cache` command from
+`huggingface-cli` tool. This command scans the cache and prints a report with information
+like repo id, repo type, disk usage, refs and full local path.
+
+The snippet below shows a scan report in a folder in which 4 models and 2 datasets are
+cached.
+
+```text
+➜ huggingface-cli scan-cache       
+REPO ID                     REPO TYPE SIZE ON DISK NB FILES REFS                LOCAL PATH                                                                
+--------------------------- --------- ------------ -------- ------------------- ------------------------------------------------------------------------- 
+glue                        dataset         116.3K       15 2.4.0, main, 1.17.0 /Users/lucain/.cache/huggingface/hub/datasets--glue                       
+google/fleurs               dataset          64.9M        6 refs/pr/1, main     /Users/lucain/.cache/huggingface/hub/datasets--google--fleurs             
+Jean-Baptiste/camembert-ner model           441.0M        7 main                /Users/lucain/.cache/huggingface/hub/models--Jean-Baptiste--camembert-ner 
+bert-base-cased             model             1.9G       13 main                /Users/lucain/.cache/huggingface/hub/models--bert-base-cased              
+t5-base                     model            10.1K        3 main                /Users/lucain/.cache/huggingface/hub/models--t5-base                      
+t5-small                    model           970.7M       11 refs/pr/1, main     /Users/lucain/.cache/huggingface/hub/models--t5-small                     
+
+Done in 0.0s. Scanned 6 repo(s) for a total of 3.4G.
+Got 1 error(s) while scanning. Use -vvv to print details.
```

Comment:
I've used `-vvv` to skip the `-vv` in case we need an extra depth in the future (since I don't know how this tool will evolve).
And also because it doesn't matter so much in my opinion. I'm fine to switch to `-vv` if you find it cleaner though